postgresql.Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM docker-registry.internal.remerge.net/python:3-alpine AS base
+
+# Image to Build Dependencies
+FROM base AS builder
+
+WORKDIR /usr/src/app
+
+COPY ./requirements.txt /usr/src/app
+
+# Build Dependencies
+RUN apk --no-cache add postgresql-dev gcc musl-dev libffi-dev openssl-dev libxml2-dev libxslt-dev file llvm-dev make g++ cargo rust
+
+# Python Dependencies
+RUN pip install --no-cache-dir --prefix=/install wheel cryptography gunicorn psycopg2-binary
+RUN pip install --no-cache-dir --prefix=/install -r requirements.txt
+
+# Runtime Environment Image
+FROM base
+
+ARG UID=1000
+ARG GID=1000
```

Comment:
Why not use the `USER` directive?

Reply:
In order for the `chown` later down  - but either way this file doesn't belong in this PR